src/main/java/com/landa44/simplerpg/attribute/Attribute.java

```diff
@@ -0,0 +1,56 @@
+package com.landa44.simplerpg.attribute;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+
+/**
+ * Attribute class represent the attributes that a character can have.
+ */
+@Entity
+@Table(name = "attributes")
+public class Attribute {
+    @Id
+    @SequenceGenerator(
+        name = "attribute_sequence",
+        sequenceName = "attribute_sequence",
+        allocationSize = 1
+    )
+    @GeneratedValue(
+        strategy = GenerationType.SEQUENCE,
+        generator = "attribute_sequence"
+    )
+    private Long id;
+    private String name;
+
+    public Attribute() {
+    }
+
+    public Attribute(String name) {
+        this.name = name;
+    }
+
+    public Attribute(Long id, String name) {
+        this.id = id;
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public String toString() {
+        return "Attribute{"
+            + "id=" + id
+            + ", name='" + name + '\''
+            + '}';
```

Comment:
String's concatenation in general is expensive, as Strings in java are immutable, the use of the plus operator(+) creates a new String every time and copies the values of both Strings to the new String, so, maybe it would be better to use something like StringBuilder or StringBuffer where you can append all the strings that you want, and when you are ready, build the String in a lazy way and copy all the values to the new String, so there will be only one new String allocated in memory, the con here is that is not as readable as using the plus operator, but it's important to have it in mind

You can read this article for more information: https://redfin.engineering/java-string-concatenation-which-way-is-best-8f590a7d22a8